src/Grid.ts

```diff
@@ -299,12 +299,34 @@ abstract class Grid<Options extends GridOptions = GridOptions> extends Component
     const mounted: GridItem[] = updated.filter((item) => item.mountState !== MOUNT_STATE.MOUNTED);
     const moreUpdated: GridItem[] = [];
 
+    mounted.filter((item) => {
+      if (!item.hasTransition) {
+        const element = item.element!;
+        const transitionDuration = parseFloat(getComputedStyle(element).transitionDuration);
+
+        if (transitionDuration > 0) {
+          item.hasTransition = true;
+          item.transitionDuration = element.style.transitionDuration;
+          return true;
+        }
+      }
+      return false;
+    }).forEach((item) => {
+      item.element!.style.transitionDuration = "0s";
+    });
     this._im?.destroy();
     this._im = new ImReady({
       prefix: this.options.attributePrefix,
     }).on("preReadyElement", (e) => {
       updated[e.index].updateState = UPDATE_STATE.WAIT_LOADING;
     }).on("preReady", () => {
+      mounted.forEach((item) => {
+        if (item.hasTransition) {
+          const element = item.element!;
+
+          element.style.transitionDuration = item.transitionDuration;
```

Comment:
Looks like it's restoring the whole original CSS text in line 339, while it's only restoring transition duration here.
Is it okay to not restore the CSS text here?

Reply:
1. When an item is removed, there is no need to restore it.

2. Only the element was removed. The element is cached, and hasTransition and transitionDuration values are also maintained. This needs to be handled by InfiniteGrid.
3.